### DISCUSSION TITLE
CSS Battle #44 – Stripes

### DISCUSSION THREAD

**Discussion by Narigo** [Virtual-Coffee/virtualcoffee.io#854] [CSS Battles]
### Link to battle:

[Let's battle! ⚔️](https://cssbattle.dev/play/44)

### Copy the code block below to format your comment on the discussion thread:

````md
<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>
<!-- have to be followed by an empty line! -->

```html
your cssbattles code here!
```

</details>
````

### What others will see:

This will result in a nice hidden bit like so:

<details>
<summary>Code Source  – <strong>score {character count}</strong></summary>

```html
your cssbattles code here!
```
</details>

**Comment by Narigo** [comment]
<details>
<summary>Solution with gradients</summary>

```html
<style>
  body {
    background: 
      radial-gradient(circle, #1A4341 50vh, transparent 0) -50vw,
      linear-gradient(#1A4341 0 60px, transparent 0 240px, #1A4341 0),
      repeating-linear-gradient(#1A4341 0 20px, #F3AC3C 0 40px)
    ;
  }
</style>
```
</details>

**Reply by danieltott** [reply]
Oh man I LOVE using the normal background repeat to get both circles. Jealous I didn't think of that!

**Comment by danieltott** [comment]
Here's my non-optimized solution, using some layered gradients as well

<details>
<summary>Dan's first solution</summary>

```html
<style>
* {
  background: #1a4341;
}
* > * {
  margin: 60 120;
  background: radial-gradient(circle at 280px, #1a4341 150px, #0000 0),
    radial-gradient(circle at -120px, #1a4341 150px, #0000 0),
    repeating-linear-gradient(#f3ac3c 0 20px, #0000 20px 40px);
}
</style>
```
</details>
